### DISCUSSION TITLE
Erreur de réservation après MAJ 3.4.0 -> 4.1

### DISCUSSION THREAD

**Discussion by Faihowar** [JeromeDevome/GRR#269] [Q&A]
Bonjour,

je viens de mettre à jour en test GRR en passant de la version 3.4 à la version 4.1 RC1. Je n'ai pas eu de problème lors de la mise à jour de la BDD et tout semble fonctionné correctement sauf la réservation. Dès que j'essaye de valider une nouvelle réservation, j'ai le message suivant : (error_end_date) 
Si quelqu'un a une idée pour une solution, je suis preneur
Merci d'avance

**Comment by ynaessens** [comment]
Bonjour,
cette erreur doit se déclencher lorsque vous entrez une date de fin antérieure à la date de début.
Pouvez-vous préciser le contexte de votre échec : date de début, de fin, avec les horaires, ou les créneaux visés ?
Cordialement,
YN
